webroot/README.md

```diff
@@ -44,25 +44,46 @@
             - `noindex,nofollow`
```

Comment:
I _think_ you are using Vuex for frontend VueJS stores. My understanding is that Pinia is the new standard for Vue. Any particular reason you chose vuex over pinia?